core/modules/system/system.admin.inc

```diff
@@ -740,6 +746,12 @@ function system_sort_themes($a, $b) {
   if ($b->is_default) {
     return 1;
   }
+  if ($a->is_admin) {
+    return -1;
+  }
+  if ($b->is_admin) {
+    return 1;
+  }
```

Comment:
Can we please merge those with the conditionals right above with `||`s?

Reply:
Yes, done. 🙏